can: DeathPactException in HttpHostConnector

From the ML ()https://groups.google.com/d/msg/spray-user/pxNnNW4iNXg/kGtFGp1QRRgJ:

```
I am seeing log messages:

INFO   | jvm 1    | 2013/11/17 10:34:30 | [ERROR] [11/17/2013 10:34:30.203] 
[rendering-actorz-akka.actor.default-dispatcher-1562] 
[akka://rendering-actorz/user
/IO-HTTP/host-connector-183] Monitored actor 
[Actor[akka://rendering-actorz/user/IO-HTTP/host-connector-183/6]] 
terminated
INFO   | jvm 1    | 2013/11/17 10:34:30 | 
akka.actor.DeathPactException:Monitored actor 
[Actor[akka://rendering-actorz/user/IO-HTTP/host-connector-183/6]] 
terminated

As far as I understand this happens when client connection to certain host 
is not used for long time. Am I correct or I harms my application somehow, 
so I should worry about this? :)
```

--- a/spray-can/src/main/scala/spray/can/client/HttpHostConnector.scala
+++ b/spray-can/src/main/scala/spray/can/client/HttpHostConnector.scala
@@ -100,6 +100,8 @@ private[can] class HttpHostConnector(normalizedSetup: Http.HostConnectorSetup, c
         context.become { // after having initiated our shutdown we must be bounce all requests
           case request: HttpRequest ⇒ context.parent.forward(request -> normalizedSetup)
           case _: Http.CloseAll     ⇒ sender ! Http.ClosedAll; context.stop(self)
+          case _: Terminated ⇒ // ignore, can happen if the last slot has sent us a `DemandIdleShutdown` and
+                               // a `ReceiveTimeout` is coming in before the `Terminated` event from the last slot
         }
       }
   }

